lizmap/install/installer.php

```diff
@@ -4,4 +4,6 @@
 
 require __DIR__.'/../application.init.php';
 
+jFile::copyDirectoryContent(__DIR__.'/../vendor/jelix/jelix/lib/jelix-www', __DIR__.'/../www/assets/jelix');
```

Comment:
No. The jelix module installer already does this copy... 

I guess the main issue we have to fix is in the build step of the zip package, where we should copy the directory into the build directory.

Reply:
Thanks I have removed this line